docs/Installation/Installation.md

```diff
@@ -205,7 +205,9 @@ To use Singularity you must:
    Singularity instructions on setting up additional [bind
    points](https://sylabs.io/guides/3.7/user-guide/bind_paths_and_mounts.html)
    (version 3.7. For other versions, see the [docs](https://sylabs.io/docs/)).
-   Once inside the WORKDIR, clone SpECTRE into `WORKDIR/SPECTRE_ROOT`.
+   Once inside the WORKDIR, clone SpECTRE into `WORKDIR/SPECTRE_HOME`. The
```

Comment:
Why change away from `SPECTRE_ROOT`? The rest of the documentation above uses `SPECTRE_ROOT`

Reply:
At the "Using docker" section above, `SPECTRE_ROOT` was used to denote the full directory name rather than the name of a folder inside a certain `WORKDIR`. I thought it would be less confusing to make a distinction between those. Would it be better to stick with just `SPECTRE_ROOT`?

Reply:
How about rephrasing from line 208 like this : 

> Once inside the WORKDIR, clone SpECTRE and let's call the source directory SPECTRE_ROOT. For instance, if you run `git clone` inside WORKDIR, it would be `WORKDIR/spectre`. You will find `docs`, `src`, `support`, `tests` etc. inside SPECTRE_ROOT.

so just stick to `SPECTRE_ROOT` throughout the documentation?

Reply:
I think `SPECTRE_ROOT` should refer to the root directory as an absolute path is clearest. Ultimately, people need to get into the root directory, whether they want to do that via an absolute or relative `cd` is up to them.

Reply:
Yep. I will stick to its definition as an absolute path.

Reply:
I see. I will stick to the original statement `WORKDIR/SPECTRE_ROOT`.